app/assets/javascripts/admin/controllers/admin-logs-staff-action-logs.js.es6

```diff
@@ -1,47 +1,75 @@
 import { exportEntity } from "discourse/lib/export-csv";
 import { outputExportResult } from "discourse/lib/export-result";
 import StaffActionLog from "admin/models/staff-action-log";
-import computed from "ember-addons/ember-computed-decorators";
+import {
+  default as computed,
+  observes,
+  on
+} from "ember-addons/ember-computed-decorators";
 
 export default Ember.Controller.extend({
   loading: false,
   filters: null,
   userHistoryActions: [],
+  model: [],
```

Comment:
This is not recommended to do in Ember because all instances point at the same object (`[]`). Instead you should use `null` and initialize the value in another place such as the route.